### DISCUSSION TITLE
No response for cmd `glow -`

### DISCUSSION THREAD

**Discussion by Abeautifulsnow** [charmbracelet/glow#395] [General]
I'm not very sure this is a bug or any other flaw. 

![image](https://user-images.githubusercontent.com/28704977/196846084-8b563022-76af-4dc5-84bc-027c0ccff19c.png)

The command has been activated for over ten minutes, but there is still no response, no output...>_<||

**Comment by muesli** [comment]
`glow -` reads markdown from `stdin`. You typically use this when you pipe data to glow, for example:

```bash
cat README.md | glow -
```

If you use it like you do, there's no data being sent to glow, so `stdin` is your keyboard input. You can start typing markdown now and when you're finished press `Ctrl-D` to signal the end of your input. Glow will then proceed rendering your input.

**Reply by Abeautifulsnow** [reply]
Thank you, I know how to use it now. 

And I have a little suggestion that can we feed back a usage message and tell who is using it that knows how to use this cmd, but not nothing is returned?

**Reply by muesli** [reply]
That's tricky. `glow -` is typically used as part of a pipe, in combination with other tools. In case input is already supplied by the pipe, the message is probably just confusing and shouldn't show up. If the output is connected to another tool in the pipe, the message might be unexpected as well.
